tests/loopbacked_tests.rs

```diff
@@ -33,19 +33,53 @@ use util::simple_tests::test_setup;
 use util::simple_tests::test_teardown;
 use util::simple_tests::test_thinpool_device;
 
+pub struct LoopTestDev {
+    ld: LoopDevice,
+}
+
+impl LoopTestDev {
+    pub fn new(lc: &LoopControl, path: &std::path::PathBuf) -> LoopTestDev {
```

Comment:
Make type of path ```AsRef<Path>```, see https://github.com/mulkieran/loopdev/pull/1/commits/24026db7d7fc7f242f14783eb007c5b28e393017 for explanation.